Formula/quartz-wm.rb

```diff
@@ -0,0 +1,50 @@
+class QuartzWm < Formula
+  desc "XQuartz window-manager"
+  homepage "https://gitlab.freedesktop.org/xorg/app/quartz-wm"
+  url "https://gitlab.freedesktop.org/xorg/app/quartz-wm/-/archive/quartz-wm-1.3.2/quartz-wm-quartz-wm-1.3.2.tar.bz2"
+  sha256 "dd185c7d7db9166c1e5ff2808bb5f6bc1b85321fafc1d3a3cf9968fb29c992ef"
+  license "APSL-2.0"
+
+  depends_on "autoconf"    => :build
+  depends_on "automake"    => :build
+  depends_on "libtool"     => :build
+  depends_on "pkg-config"  => :build
+  depends_on "util-macros" => :build
+  depends_on "xorg-server" => :test
+
+  depends_on "libapplewm"
+  depends_on "libxinerama"
+  depends_on "libxrandr"
+  depends_on :macos
+  depends_on "pixman"
+
+  def install
+    # Fix build on Monterey
+    # See https://trac.macports.org/ticket/63355
+    if MacOS.version >= :monterey
+      inreplace "src/quartz-wm.h", /(?=#undef Picture)/,
+              "#include <ApplicationServices/ApplicationServices.h>\n"
+    end
```

Comment:
Could we apply https://gitlab.freedesktop.org/xorg/app/quartz-wm/-/commit/941dd6ca2e18955d063501f1d0d239646418d416 so it's more obvious that there's a patch upstream?

Reply:
This causes audit failed, hoping there is new tag, the latest tag is 6 years ago... `XQuartz` always use the `HEAD` in submodule.

Reply:
> Could we apply https://gitlab.freedesktop.org/xorg/app/quartz-wm/-/commit/941dd6ca2e18955d063501f1d0d239646418d416 so it's more obvious that there's a patch upstream?

Use the revision form `XQuartz` now (the latest revision).